homeassistant/loader.py

```diff
@@ -83,7 +83,11 @@ def get_platform(hass,  # type: HomeAssistant
     """
     # If the platform has a component, we will limit the platform loading path
     # to be the same source (custom/built-in).
-    component = _load_file(hass, platform_name, LOOKUP_PATHS)
+    if domain not in ['automation', 'mqtt', 'telegram_bot']:
```

Comment:
This is not good. You are now allowing partial overlays for `automation`,`mqtt` and `telegram_bot`. That is not allowed.

Reply:
Oh I see now what's being done. ok.